generated/artifacts/openapi-yaml/.openapi-generator/VERSION

```diff
@@ -0,0 +1 @@
+7.2.0
```

Comment:
This is just a way used by the openapi generator itself to keep track of its own version at generation time. I think we should leave that as it's, anyway it's going to be overridden in case a different version is used to re-generate artifacts.

Reply:
I'll _git-ignore_ this file too, anyway added value is quite small.